docs/reference/metadata.md

```diff
@@ -112,6 +113,8 @@ Example:
 }
 ```
 
+Comment: Why again did we split the internal identiier into `id` and `pid` and not just `pid` with subentries `id`, `pk`, `status`.
```

Comment:
Minor: Just remove this comment, and the PR can be merged.

Reply:
The reason is because we want the JSON document in the RESTAPI to have a top-level ``id`` key (so it's easily understandable what is a document's primary id). Because we don't want differences between internal/external JSON representation, we then splut the extra properties into a separate key

Reply:
Oups, that was a leftover from a long time ago that I forgot was there! Removing.